Update documentation (#12)
elasticpath/facebook-chat: chatbot/README.md

At this point the chatbot is up and running!

> ### Docker

Reviewer: I'd prefer grouping the `Docker` section with the `Deploy the application` section to keep things together if they're related (for production deployments). 
If the `Docker` section is meant for production, lets rename it to `Deploy for production`, similar to the Reference Storefront's layout.

Author: Fixed

The application contains a `Docker` folder that contains a Dockerfile and a Shell script. The Dockerfile uses a regular NodeJS image and contains multiple arguments that you can populate using the `--build-args` argument when building the image.
These Docker arguments are used in the Shell script to replace the constants in the code, this way, the chatbot is easily deployable and doesn't require to modify directly the code.

|  Parameter| Importance|Type|Description|
|--|--|--|--|
|`PAGE_ACCESS_TOKEN`| Required| String| The token that is provided by Facebook on (developers.facebook.com) for each application.|
|`VERIFY_TOKEN`| Required| String| The token that you have to generate and pass to Facebook when configuring the webhooks.|

> |`SCOPE`|Required|String| The Cortex scope (store) that the chatbot is going to use.|

Reviewer: ```suggestion
|`SCOPE`|Required|String| Name of the store from which Cortex retrieves data.|
```

> |`S3_URL`|Required|String| The AWS S3 URL that the application is using to fetch images.|

Reviewer: ```suggestion
|`S3_URL`|Required|String| The URL that consists of the path to images hosted on an external CMS.|
```

If possible, let's also try to describe how the url should be structured. Similar to how we describe it for the Reference Storefront.
